site/components/map/StopsMap.tsx

```diff
@@ -116,6 +116,8 @@ const Map = ({
                         ? { stopTypes: ["MET", "PLT"] }
                         : vehicleMode === Modes.ferry || vehicleMode === VehicleMode.ferryService
                         ? { stopTypes: ["FER", "FBT"] }
+                        : vehicleMode === Modes.rail
```

Comment:
I think these lines of code also need adding to /create-consequence-stops/[disruptionId]/[consequenceIndex].page.tsx to allow Rail stations to be selected via the drop down.

Reply:
done